src/lib/discord/commands/q.ts

```diff
@@ -0,0 +1,133 @@
+import { EmbedBuilder, ButtonStyle } from 'discord.js'
+import {
+  SlashCommandBuilder,
+  ActionRowBuilder,
+  ButtonBuilder,
+} from '@discordjs/builders'
+import { prisma } from '$lib/db'
+import type {
+  ButtonInteraction,
+  ChatInputCommandInteraction,
+  InteractionReplyOptions,
+  TextBasedChannel,
+} from 'discord.js'
+import type { Question } from '@prisma/client'
+
+function epoch(date: Date): number {
+  return Math.floor(date.getTime() / 1000)
+}
+
+function createDiscordTimestamp(date: Date): string {
+  return `<t:${epoch(date)}:R>`
+}
+
+function createQuestionEmbedFields(questions: Question[]) {
+  return questions.map((question) => ({
+    name: `${question.isSolved ? '✅' : '﹖'} ${createDiscordTimestamp(
```

Comment:
It looks like we're only pulling unsolved questions from the DB, right?

Reply:
ah more remnants. Can you think of a use case where we would want solved questions? should this be removed